src/endpoints/createFetcher.js

```diff
@@ -70,7 +70,7 @@ function createFetcher (endpoint) {
 
       if (!payload.ready()) {
         if (!this.props.hideSpinner) {
-          return <Spinner className={cx(style.loading, this.props.className)} name='line-scale-party'/>
+          return <Spinner className={cx(style.loading, this.props.className)} fadeIn="quarter" name='line-scale-party'/>
```

Comment:
Revert this: https://github.com/KyleAMathews/react-spinkit#fades-in-spinners-after-one-second

Reply:
C'mon noone wants to wait for a whole second before he gets some reaction imo :D

Reply:
yeah but research has been done :P and it makes sense